rust_rocks.md

```diff
@@ -0,0 +1,13 @@
+# Rust Rocks!
+
+Consider this the unofficial Rust sales pitch.
+
+## Types
+
+Rust has strong static types. This can be very different than what you are used to if you are coming from a loosely-typed language. Don't worry, though. Once you get used to them, you'll find the types actually make your life easier. Why? Because you are making implicit assumptions explicit.
+
+TODO: Show an example that illustrates the importance of data types.
```

Comment:
This seems like a good intro to the new type pattern: https://rust-unofficial.github.io/patterns/patterns/newtype.html

Reply:
Done